.github/workflows/linters.yml

```diff
@@ -0,0 +1,8 @@
+name: Linters
+on: [ push, pull_request ]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: chartboost/ruff-action@v1
```

Comment:
Let's use pre-commit action instead: https://github.com/pre-commit/action (ruff is already included)

Reply:
Done